### PR TITLE
Sync with the rke2 information about embedded registry

### PR DESCRIPTION
**Pull request by manuelbuil**
This PR:
* Fills the gaps with the RKE2 docs https://github.com/rancher/rke2-docs/pull/296
* Adds the GA information

### CODE REVIEW

File: docs/installation/airgap.md [hunk 1]
```diff
@@ -43,7 +44,7 @@ This method requires you to manually deploy the necessary images to each node, a
 ### Embedded Registry Mirror
 
 :::info Version Gate
-The Embedded Registry Mirror is available as an experimental feature as of January 2024 releases: v1.26.13+k3s1, v1.27.10+k3s1, v1.28.6+k3s1, v1.29.1+k3s1
+The Embedded Registry Mirror is available as an experimental feature as of January 2024 releases: v1.26.13+k3s1, v1.27.10+k3s1, v1.28.6+k3s1, v1.29.1+k3s1 and GA as of December 2024 releases: v1.29.12+k3s1, v1.30.8+k3s1, v1.31.4+k3s1
 :::
 
 K3s includes an embedded distributed OCI-compliant registry mirror.
```
Comment (dereknola) [inline docs/installation/airgap.md:50]: Can we just shorten this section and point to actual docs.

File: docs/installation/airgap.md [hunk 2]
```diff
@@ -43,7 +44,7 @@ This method requires you to manually deploy the necessary images to each node, a
 ### Embedded Registry Mirror
 
 :::info Version Gate
-The Embedded Registry Mirror is available as an experimental feature as of January 2024 releases: v1.26.13+k3s1, v1.27.10+k3s1, v1.28.6+k3s1, v1.29.1+k3s1
+The Embedded Registry Mirror is available as an experimental feature as of January 2024 releases: v1.26.13+k3s1, v1.27.10+k3s1, v1.28.6+k3s1, v1.29.1+k3s1 and GA as of December 2024 releases: v1.29.12+k3s1, v1.30.8+k3s1, v1.31.4+k3s1
```
Comment (dereknola) [inline docs/installation/airgap.md:47]: I would just rip this out entirely, and just point to the docs that have that info.

**Review by dereknola — CHANGES_REQUESTED** [review]

File: docs/installation/airgap.md (hunk 1 above)
Comment (manuelbuil) [reply docs/installation/airgap.md:50]: Sorry, what docs do you mean?

File: docs/installation/airgap.md (hunk 2 above)
Comment (manuelbuil) [reply docs/installation/airgap.md:47]: I agree. Let me remove it and we have the information in the embedded-registry doc

**Review by dereknola — APPROVED** [review]

**Review by brandond — APPROVED** [review]

**Pull request merged**